techmatters/flex-plugins
# CHI-1434: Create contact on new task.

## Description

This PR makes 2 major changes:

* The contact is created in HRM either on accepting a twilio task or starting an offline contact, then updated with the full details later
* The redux store now holds all contacts under the 'existingContacts' store keyed by contactId, the old 'tasks' store keyed on taskSid previously used for unsaved contacts is now gone

To do in a follow up PR - align some of the changes in here with the refactor @acedeywin did concurrently to do contact updates via redux

### Checklist

### Related Issues
CHI-1433, CHI-1434

### Verification steps

* The correct version of HRM needs to be deployed before verifying this build, talk to @stephenhand 
* Regression test existing functionality
* Check that you get your forms back if you refresh partway through a new contact (except for the changes on the last tab)
